8_reference/80_relay_api.txt

```diff
@@ -41,6 +41,7 @@ This API provides the following methods:
 The general parameters are:
 
 * `keep_output` = *true* or *false*: Should the agent output be returned (default: *false*)
+* `asynchronous` = *true* or *false*: If `keep_output` is true, should the output be returned as it is produced (contrary to waiting for the end of the run). If it is false, should the call return just after starting the agent, without waiting for the end. (default: *false*)
```

Comment:
In all cases, even if we don't keep_output,  if asynchronous is 'false' we wait for return of 'query' or 'command', asynchronous without output returns directly, meaning you don't have to wait to check if it has run correctly on your xxxx nodes (which can be quite long)